cli/cmd/version.go

```diff
@@ -4,24 +4,37 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/runconduit/conduit/cli/k8s"
+	"github.com/runconduit/conduit/cli/shell"
+
 	"github.com/runconduit/conduit/controller"
 	pb "github.com/runconduit/conduit/controller/gen/public"
 	"github.com/spf13/cobra"
 )
 
+const DefaultVersionString = "unavailable"
+
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print the client and server version information",
 	Long:  "Print the client and server version information.",
 	Args:  cobra.NoArgs,
 	Run: exitSilentlyOnError(func(cmd *cobra.Command, args []string) error {
-		fmt.Println("Client version: " + controller.Version)
```

Comment:
I'm running this from a Docker container, perhaps that's why I see no output?

If either `k8s.MakeK8sAPi()` or `newApiClient()` fail, it prints nothing. I think we should print the client version no matter what, and then perhaps let the user know what went wrong.